containerized_ml_models/api/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM continuumio/anaconda3
+
+COPY model.pkl requirements.txt api.py ./
+
+EXPOSE 8000
+
+RUN conda update -n base conda && \
+    conda create -n venv python=3.9 \
+    conda install scikit-learn \
```

Comment:
There is no need to install scikit-learn manually, you are passing it again to pip via requirements.txt